api/server/fns_delete.go

```diff
@@ -0,0 +1,29 @@
+package server
+
+import (
+	"net/http"
+
+	"github.com/fnproject/fn/api"
+	"github.com/gin-gonic/gin"
+)
+
+func (s *Server) handleFnsDelete(c *gin.Context) {
+	ctx := c.Request.Context()
+
+	fn := c.Param(api.Fn)
+	appName := c.Param(api.CApp)
+
+	appID, err := s.datastore.GetAppID(ctx, appName)
```

Comment:
this should already be set as a param by https://github.com/fnproject/fn/blob/master/api/server/gin_middlewares.go#L172 if we want to use `MustGet` here, it saves an additional lookup. route is this way - am not sure if we are diverging from the routes way, which could be ok too - just throwing this out.